alacritty/src/window.rs

```diff
@@ -158,6 +162,10 @@ impl Window {
                 if let Some(parent_window_id) = config.window.embed {
                     x_embed_window(windowed_context.window(), parent_window_id);
                 }
+            } else {
+                let theme = AlacrittyWaylandTheme::new(&config.colors);
+
+                windowed_context.window().set_wayland_theme(theme);
```

Comment:
```suggestion
                let theme = AlacrittyWaylandTheme::new(&config.colors);
                windowed_context.window().set_wayland_theme(theme);
```

This is mostly just a suggestion based on my preference. Doesn't really matter either way.